modules/cluster-wide-proxy-preqs.adoc

```diff
@@ -0,0 +1,72 @@
+[id="cluster-wide-proxy-prereqs_{context}"]
+= Prerequisites for configuring a cluster-wide proxy
+
+To configure a cluster-wide proxy, you must meet the following requirements. These requirements are valid for both fresh installation and post installation proxy configuration.
+
+[id="cluster-wide-proxy-general-prereqs_{context}"]
+== General requirements
+
+* You are the cluster owner.
+* Your account has sufficient privileges.
+* You have added the `ec2.<region>.amazonaws.com`, `elasticloadbalancing.<region>.amazonaws.com`, and `s3.<region>.amazonaws.com` endpoints to your virtual private cloud (VPC) endpoint. These endpoints are required to complete requests from the nodes to the AWS EC2 API. Because the proxy works on the container level, not the node level, you must route these requests to the AWS EC2 API through the AWS private network. Adding the public IP address of the EC2 API to your allowlist in your proxy server is not sufficient.
+ifdef::openshift-rosa[]
+* You have the `rosa` CLI installed and configured.
+endif::[]
+ifdef::openshift-dedicated[]
+* You must have a Customer Cloud Subscription (CCS) cluster with a VPC that the proxy can access.
+* You have the `ocm` CLI installed and configured.
+endif::[]
+
+[id="cluster-wide-proxy-network-prereqs_{context}"]
+== Network requirements
+
+* If your proxy re-encyrpts egress traffic, you must create exclusions to the domain and port combinations. The following table offers guidance into these exceptions.
+** Allowlist the following OpenShift URLs for re-encryption.
++
+[cols="6,1,6",options="header"]
+|===
+|Address | Protocol/Port | Function
+|`observatorium-mst.api.openshift.com`
+|https/443
+|Required. Used for Managed OpenShift-specific telemetry.
+
+|`sso.redhat.com`
+|https/443
+|The https://cloud.redhat.com/openshift site uses authentication from sso.redhat.com to download the cluster pull secret and use Red Hat SaaS solutions to facilitate monitoring of your subscriptions, cluster inventory, and chargeback reporting.
+|===
++
+** Allowlist the following site reliability engineering (SRE) and management URL for re-encryption.
++
+[cols="6,1,6",options="header"]
+|===
+|Address | Protocol/Port | Function
+|`*.osdsecuritylogs.splunkcloud.com`
+
+**OR**
+
+`inputs1.osdsecuritylogs.splunkcloud.com`
+`inputs2.osdsecuritylogs.splunkcloud.com`
+`inputs4.osdsecuritylogs.splunkcloud.com`
+`inputs5.osdsecuritylogs.splunkcloud.com`
+`inputs6.osdsecuritylogs.splunkcloud.com`
+`inputs7.osdsecuritylogs.splunkcloud.com`
+`inputs8.osdsecuritylogs.splunkcloud.com`
+`inputs9.osdsecuritylogs.splunkcloud.com`
+`inputs10.osdsecuritylogs.splunkcloud.com`
+`inputs11.osdsecuritylogs.splunkcloud.com`
+`inputs12.osdsecuritylogs.splunkcloud.com`
+`inputs13.osdsecuritylogs.splunkcloud.com`
+`inputs14.osdsecuritylogs.splunkcloud.com`
+`inputs15.osdsecuritylogs.splunkcloud.com`
+|tcp/9997
+|Used by the splunk-forwarder-operator as a logging forwarding endpoint to be used by Red Hat SRE for log-based alerting.
```

Comment:
s/logging forwarding endpoint/log forwarding endpoint ? When I was on the logging team, the term was log forwarding. Maybe it changed?